README.md

```diff
@@ -27,7 +27,13 @@ This is the backend server for a web application for tracking the placement acti
     ```sh
     npm start
     ```
-
+### Create a .env file in the root directory of the project, and add the following environment variables:
+ ```sh
```

Comment:
```suggestion
 ```env
```

Reply:
Got it

Reply:
@Ashrockzzz2003 Please check now

Reply:
Should I add some extra stuff like a guide??

Reply:
Change it to `env` the code block language.